dcicutils/misc_utils.py

```diff
@@ -802,3 +812,14 @@ def __repr__(self):
         return 'CachedField %s with update function %s on timeout %s' % (
             self.name, self._update_function, self.timeout
         )
+
+
+def make_counter(start=0):
```

Comment:
Might be worth adding `, step=1` here in the params, and later saying `storage[0] += step`. Would require more test cases. We could add this later, but it's such an obvious detail this might be the better time.

For number theory fans, it's worth noting that if you add a `step`, you can also do odd things like
```
make_counter([], step=['foo'])
```
or
```
make_counter("", step=". ")
```
not that I imagine anyone will.